src/core/receiver/gnss_block_factory.cc

```diff
@@ -39,6 +39,9 @@
 #include "acquisition_interface.h"  // for AcquisitionInterface
 #include "array_signal_conditioner.h"
 #include "beamformer_filter.h"
+#include "beidou_b1c_dll_pll_tracking.h"
+#include "beidou_b1cd_pcps_acquisition.h"
```

Comment:
Maybe this file should be removed, the beidou_b1c_pcps_acquisition takes care of the work now